clio/example-third-party-application
# Remove Unused Database

Removed the database as it wasn't used. As a result, had to remove active record as it relies on a database connection.

**How do we test this?**
Ensure you can still successfully start the app, view your profile, all matters and a single matter.
